File: SharedProject1/GetInstanceRequest.cs
using System.Xml;
using System.Xml.Serialization;

[XmlRoot("GetInstanceData", Namespace = "http://ffiec.gov/cdr/services/")]
public class GetInstanceRequest
{
	public const string DateFormat = "yyyy-MM-dd";
	[XmlElement("userName")]
	public string Username { get; set; }
	[XmlElement("password")]
	public string Password { get; set; }
	[XmlElement("dataSeriesName")]
	public string DataSeriesName { get; set; }
	[XmlElement("reportingPeriodEndDate")]
	public DateTime ReportingPeriodEndDate { get; set; }
	[XmlElement("id_rssd")]
	public string IdRssd { get; set; }
	[XmlElement("numberOfPriorPeriods")]
	public int NumberOfPriorPeriods { get; set; }
}

File: VB/SharedProject1/GetInstanceRequest.vb
Imports System.Xml
Imports System.Xml.Serialization

<XmlRoot("GetInstanceData", [Namespace]:="http://ffiec.gov/cdr/services/")>
Public Class GetInstanceRequest
	Public Const DateFormat As String = "yyyy-MM-dd"
	<XmlElement("userName")>
	Public Property Username As String
	<XmlElement("password")>
	Public Property Password As String
	<XmlElement("dataSeriesName")>
	Public Property DataSeriesName As String
	<XmlElement("reportingPeriodEndDate")>
	Public Property ReportingPeriodEndDate As DateTime
	<XmlElement("id_rssd")>
	Public Property IdRssd As String
	<XmlElement("numberOfPriorPeriods")>
	Public Property NumberOfPriorPeriods As Integer
End Class
